context/app/static/js/theme.js

```diff
@@ -0,0 +1,79 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+import "typeface-inter";
+
+// default HuBMAP color and font theme
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: '#444A65',
+    },
+    secondary: {
+      main: '#636363',
+    },
+    error: {
+      main: '#DA348A',
+    },
+    warning: {
+      main: '#D25435',
+    },
+    info: {
+      main: '#3781D1',
+    },
+    success: {
+      main: '#6C8938' //'#9BC551'
+    },
+    type: "light"
```

Comment:
I remember you thought some of the colors in the palette were problematic... Is that still an open question, or is Nils on board with what I see here?

Reply:
The warning color is still in question but was not utilized in this pull request.